elixir/lib/cli.ex

```diff
@@ -1,7 +1,9 @@
 defmodule GithubRepoCloner.CLI do
+  alias GithubRepoCloner.Cloner
+  alias GithubRepoCloner.PageIterator
+
   def main(args \\ []) do
-    args
-    |> List.first
-    |> GithubRepoCloner.Cloner.clone
+    username = List.first(args)
+    PageIterator.repeat(&Cloner.clone_page/1, %{username: username, page: 1})
```

Comment:
I think the name should change to `clone_pages` ?

Reply:
Might wrap this in a new `Cloner.clone_pages` function. CLI will use `Cloner.clone_pages`.